G/GANAK/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GANAK"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/meelgroup/ganak.git", "36fa4b35c8e3cb588a3a76a87624d97ae7f18285")
+    DirectorySource("bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ganak/
+
+# add all dependencies for linking without 'allow undefined'
+atomic_patch -p1 ../patches/alllibs.patch
+
+# make sure char in LogTable is signed on all platforms (needed for storing -1)
+atomic_patch -p1 ../patches/logtab_signed.patch
+
+mkdir build; cd build
+
+if [[ ${target} != x86_64-linux* ]]; then
+  # clhash only works on x86_64 linux platforms
+  disablepcc=-DDOPCC=OFF
+fi
+
+cmake $disablepcc \
+      -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_SKIP_BUILD_RPATH=TRUE \
+      ..
+make -j${nproc}
+install -vc ganak -Dt $bindir 
+install -vc src/libganak* -Dt $libdir
```

Comment:
What's `-c` supposed to do exactly?  My `man install` says
```
       -c     (ignored)
```

Reply:
I saw this in the sandbox:
```
sandbox:${WORKSPACE}/srcdir/ganak/build # install
BusyBox v1.34.1 (2021-11-23 00:57:35 UTC) multi-call binary.

Usage: install [-cdDsp] [-o USER] [-g GRP] [-m MODE] [-t DIR] [SOURCE]... DEST

Copy files and set attributes

	-c	Just copy (default)
```
Since this is basically my first time assembling something like this, this sounded good to me. I'll remove it.